doc/concepts/recipe.rst

```diff
@@ -17,17 +17,23 @@ building phase to provide information about cells in the model, such as:
   * **Gap junction connections** on a cell.
   * **Probes** on a cell.
 
+Recipes are structured in such a way that during the lifetime of the simulation properties
```

Comment:
This isn't strictly true.
A recipe is only needed when instantiating (constructing) a simulation: once the simulation has been constructed, the recipe can be destructed.

```
Recipes are structured to provide a consistent interface for describing each cell in the network using their global identifier (`gid`).
```

Also, the recipe provides an interface for describing connections coming "in" to the cell. The term `input events` is vague, and isn't needed.